inc/namespace.php

```diff
@@ -892,6 +892,51 @@ function filter_facet_settings( $value ) {
 	return $value;
 }
 
+/**
+ * Get the search fuzziness configuration.
+ *
+ * Fetches the "fuzziness" config value, parses and validates it returning
+ * a value that can be used in Elasticsearch queries.
+ *
+ * @return array
+ */
+function get_fuzziness() : array {
+	$fuzziness = Altis\get_config()['modules']['search']['fuzziness'] ?? 'auto:4,7';
+
+	// Handle array format.
+	if ( ! is_array( $fuzziness ) ) {
+		$fuzziness = [
+			'distance' => $fuzziness,
+		];
+	}
+
+	// Set default values.
+	$fuzziness = wp_parse_args( $fuzziness, [
+		'distance' => 'auto:4,7',
+		'prefix-length' => 1,
+		'max-expansions' => 50,
+		'transpositions' => true,
+	] );
+
+	// Validate distance value.
+	if ( ! preg_match( '/^([0-2]|auto:\d+,\d+|auto)$/', $fuzziness['distance'] ) ) {
+		trigger_error( sprintf( 'The provided fuzziness distance config option %s is invalid, defaulting to "auto:4,7"', (string) $fuzziness ), E_USER_WARNING );
```

Comment:
This doesn't need to be cast to string, as the `%s` formatter will already do this. That said, need to ignore the linter error here too.